packages/trimerge-sync-basic-client/src/WebsocketRemote.ts

```diff
@@ -1,18 +1,30 @@
-import type { ErrorCode, OnEventFn, Remote, SyncEvent } from 'trimerge-sync';
+import type {
+  ErrorCode,
+  OnEventFn,
+  Remote,
+  RemoteSyncInfo,
+  SyncEvent,
+} from 'trimerge-sync';
 
 export class WebsocketRemote<EditMetadata, Delta, PresenceState>
   implements Remote<EditMetadata, Delta, PresenceState> {
   private socket: WebSocket | undefined;
   private bufferedEvents: SyncEvent<EditMetadata, Delta, PresenceState>[] = [];
   constructor(
     auth: unknown,
-    lastSyncId: string | undefined,
+    { localStoreId, lastSyncCursor }: RemoteSyncInfo,
     private readonly onEvent: OnEventFn<EditMetadata, Delta, PresenceState>,
     websocketUrl: string,
   ) {
+    console.log(`[TRIMERGE-SYNC] Connecting to ${websocketUrl}...`);
```

Comment:
Do you want to keep these logs?

Reply:
yea, for now it's helpful. will need to think about the general logging strategy. how is the debug library in the browser?